diesel/src/pg/expression/expression_methods.rs

```diff
@@ -1301,6 +1301,129 @@ pub trait PgJsonbExpressionMethods: Expression + Sized {
     {
         Grouped(HasKeyJsonb::new(self, other.as_expression()))
     }
+    /// Creates a PostgreSQL `@>` expression.
+    ///
+    /// This operator returns whether a JSON value contains another JSON value
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # include!("../../doctest_setup.rs");
+    /// #
+    /// # table! {
+    /// #    contacts {
+    /// #        id -> Integer,
+    /// #        name -> VarChar,
+    /// #        address -> Jsonb,
+    /// #    }
+    /// # }
+    /// #
+    /// # fn main() {
+    /// #     run_test().unwrap();
+    /// # }
+    /// #
+    /// # #[cfg(feature = "serde_json")]
+    /// # fn run_test() -> QueryResult<()> {
+    /// #     use self::contacts::dsl::*;
+    /// #     let conn = &mut establish_connection();
+    /// #     conn.execute("DROP TABLE IF EXISTS contacts").unwrap();
+    /// #     conn.execute("CREATE TABLE contacts (
+    /// #         id SERIAL PRIMARY KEY,
+    /// #         name VARCHAR NOT NULL,
+    /// #         address JSONB NOT NULL
+    /// #     )").unwrap();
+    /// #
+    /// let easter_bunny_address: serde_json::Value = serde_json::json!({
+    ///     "street": "123 Carrot Road",
+    ///     "province": "Easter Island",
+    ///     "region": "Valparaíso",
+    ///     "country": "Chile",
+    ///     "postcode": "88888",
+    /// });
+    /// diesel::insert_into(contacts)
+    ///     .values((name.eq("Bunny"), address.eq(&easter_bunny_address)))
+    ///     .execute(conn)?;
+    ///
+    /// let country_chile: serde_json::Value = serde_json::json!({"country": "Chile"});
+    /// let contains_country_chile = contacts.select(address.contains(&country_chile)).get_result::<bool>(conn)?;
+    /// assert!(contains_country_chile);
+    /// #     Ok(())
+    /// # }
+    /// # #[cfg(not(feature = "serde_json"))]
+    /// # fn run_test() -> QueryResult<()> {
+    /// #     Ok(())
+    /// # }
+    /// ```
+    fn contains<T>(self, other: T) -> dsl::ContainsJsonb<Self, T>
+    where
+        T: AsExpression<Jsonb>,
+    {
+        Grouped(ContainsJsonb::new(self, other.as_expression()))
+    }
+    /// Creates a PostgreSQL `<@` expression.
+    ///
+    /// This operator returns whether a JSON value is contained by another JSON value. `foo.contains(bar)` is the
```

Comment:
```suggestion
    /// This operator checks whether left hand side JSONB value is contained by right hand side JSON value. `foo.contains(bar)` is the
```

Please break up the sentence into lines accordingly after the length changes.

Reply:
fixed